src/Symfony/Component/ErrorHandler/ErrorRenderer/CliErrorRenderer.php

```diff
@@ -108,4 +183,23 @@ private function findExceptionFilePathAndName(string $filepath): array
 
         return [$dirPath, $fileName];
     }
+
+    private function makePathRelative(string $absolutePath): string
+    {
+        $vendorPrefix = str_replace([
+            'symfony'.DIRECTORY_SEPARATOR.'error-handler'.DIRECTORY_SEPARATOR.'ErrorRenderer'.DIRECTORY_SEPARATOR.'CliErrorRenderer.php',
+            'src'.DIRECTORY_SEPARATOR.'Symfony'.DIRECTORY_SEPARATOR.'Component'.DIRECTORY_SEPARATOR.'ErrorHandler'.DIRECTORY_SEPARATOR.'ErrorRenderer'.DIRECTORY_SEPARATOR.'CliErrorRenderer.php',
+        ], '', __FILE__);
+        $projectPrefix = \dirname($vendorPrefix).DIRECTORY_SEPARATOR;
+
+        return str_replace($projectPrefix, '', $absolutePath);
```

Comment:
Instead you can use Path directly:

```php
Path::makeRelative($absolutePath, $projectPrefix);
```

Reply:
The component currently doesn't depend on `symfony/filesystem` - and probably not worth trying to include it just for this.